lib/iris/tests/test_cube_to_pp.py

```diff
@@ -266,7 +266,7 @@ def lbproc_from_pp(self, filename):
         field = next(pp_file)
         return field.lbproc
 
-    # see related comment #236
+    # see related comment line number #236
```

Comment:
This comment confused me so I thought a quick edit might prevent someone else also getting confused.
Of course the intention is to remove this `skip_biggus` soon